src/vsr/client.zig

```diff
@@ -395,11 +395,23 @@ pub fn ClientType(comptime StateMachine_: type, comptime MessageBus: type) type
             assert(eviction.header.view >= self.view);
 
             if (self.on_eviction_callback) |callback| {
-                log.err("{}: session evicted: reason={?s} (cluster_release={})", .{
-                    self.id,
-                    std.enums.tagName(vsr.Header.Eviction.Reason, eviction.header.reason),
-                    eviction.header.release,
-                });
+                const eviction_specific_log = switch (eviction.header.reason) {
+                    .client_release_too_low => " your client is too old; upgrade to a version " ++
+                        "compatible with your cluster",
+                    .client_release_too_high => " your client is too new; downgrade to the same " ++
+                        "version as your cluster",
```

Comment:
```suggestion
                    .client_release_too_low => " Your client is too old; upgrade to a version " ++
                        "compatible with your cluster",
                    .client_release_too_high => " Your client is too new; downgrade to the same " ++
                        "version as your cluster",
```

maybe? It's after `.`

Reply:
It should have a trailing full stop then too, but it feels... weird to have mixed capitalization like that.

I was really looking for a way to distinguish the information from the suggestion. Think I'll use a `-` instead.